Cargo.toml

```diff
@@ -191,3 +192,5 @@ lto       = "off"
 [profile.release]
 # https://doc.rust-lang.org/cargo/reference/profiles.html#strip
 strip = true
+# https://doc.rust-lang.org/rustc/codegen-options/index.html#split-debuginfo
+split-debuginfo = "unpacked"
```

Comment:
Doesn't this conflict with strip?

Reply:
it doesn't conflict but is probably useless, given that we strip away other symbols, playing with `gdb` would be most likely a pain here. I'll remove it for `release`, thanks